test/e2e/opentelemetry/opentelemetry.test.ts

```diff
@@ -178,13 +180,15 @@ createNextDescribe(
             Object {
               "attributes": Object {
                 "http.method": "GET",
+                "http.route": "/api/app/[param]/data/route",
                 "http.status_code": 200,
                 "http.target": "/api/app/param/data",
+                "next.route": "/api/app/[param]/data/route",
```

Comment:
what are the differences between Next and HTTP route?

Reply:
it should be the same value.

But specs don't allow using `http.route` outside SERVER span.
So I use `next.route` there and it felt weird to not include it in the top span too.